TCB/TCB.ml

```diff
@@ -334,9 +334,13 @@ let ( @ ) = ( @ )
  * See also for In: Scanf/Lexing/Parsing
  *)
 
+(*
+I don't know how to use stdin if it's shadowed -- Martin
```

Comment:
That's the point. You have to qualify with UStdlib.stdin so it's explicit you're using
the unsafe part of the stdlib (U means unsafe). I'll add a comment about it in TCB.ml